[0.524s] Invoking command in '/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake --build /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas -- -j4 -l4
[0.620s] -- Found ament_cmake: 1.3.5 (/opt/ros/humble/share/ament_cmake/cmake)
[1.120s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[1.129s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[1.148s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[1.265s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[1.370s] -- Found geometry_msgs: 4.2.3 (/opt/ros/humble/share/geometry_msgs/cmake)
[2.428s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[2.429s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name my_mas
[2.429s]   --arguments-file
[2.430s]   /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas/rosidl_adapter__arguments__my_mas.json
[2.430s]   --output-dir
[2.430s]   /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas/rosidl_adapter/my_mas
[2.431s]   --output-file
[2.431s]   /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas/rosidl_adapter/my_mas.idls)
[2.433s]   returned error code 1:
[2.433s] 
[2.433s]   Traceback (most recent call last):
[2.434s] 
[2.434s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[2.435s]       return _run_code(code, main_globals, None,
[2.435s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[2.436s]       exec(code, run_globals)
[2.436s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[2.436s]       sys.exit(main())
[2.437s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[2.437s]       abs_idl_file = convert_to_idl(
[2.438s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[2.438s]       return convert_msg_to_idl(
[2.439s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[2.439s]       msg = parse_message_string(package_name, input_file.stem, content)
[2.439s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[2.440s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[2.440s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[2.441s]       self.base_type = BaseType(
[2.441s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[2.441s]       raise InvalidResourceName(
[2.442s] 
[2.442s]   rosidl_adapter.parser.InvalidResourceName: 'Trans_global' is an invalid
[2.442s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[2.442s] 
[2.443s] Call Stack (most recent call first):
[2.443s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[2.444s]   CMakeLists.txt:14 (rosidl_generate_interfaces)
[2.444s] 
[2.444s] [0m
[2.445s] -- Configuring incomplete, errors occurred!
[2.445s] See also "/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas/CMakeFiles/CMakeOutput.log".
[2.456s] gmake: *** [Makefile:2043: cmake_check_build_system] Error 1
[2.457s] Invoked command in '/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas' returned '2': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake --build /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas -- -j4 -l4
